backend/tests/exploregens.spec.ts

```diff
@@ -60,6 +61,7 @@ import { fail } from "assert";
 describe('exploregens unit test', () => {
     let sandbox: any;
```

Comment:
type of sandbox should be SinonSandbox
type of every mock should be SinonMock

Reply:
it's the same in all our spec files!
https://github.com/SAP/yeoman-ui/blob/DEVXBUGS-8621/backend/tests/yeomanui.spec.ts#L39

Reply:
Right, it was like this until Alex Gilin found the best option. We should gradually replace all any by right type